core/templates/dev/head/app.js

```diff
@@ -34,11 +34,16 @@ var oppia = angular.module(
 // Set default headers for POST and PUT requests.
 // Add an interceptor to convert requests to strings and to log and show
 // warnings for error responses.
-oppia.config(['$interpolateProvider', '$httpProvider',
-    function($interpolateProvider, $httpProvider) {
+oppia.config(['$interpolateProvider', '$httpProvider', '$locationProvider',
+    function($interpolateProvider, $httpProvider, $locationProvider) {
   $interpolateProvider.startSymbol('<[');
   $interpolateProvider.endSymbol(']>');
 
+  $locationProvider.html5Mode(false);
+  if (window.location.pathname == '/search/find') {
```

Comment:
Angular is throwing errors if I do that because this is in the config section, so only providers are allowed.